packages/nextjs/src/config/webpack.ts

```diff
@@ -348,7 +350,7 @@ async function addSentryToEntryProperty(
  * @param platform Either "server" or "client", so that we know which file to look for
  * @returns The name of the relevant file. If no file is found, this method throws an error.
  */
-export function getUserConfigFile(projectDir: string, platform: 'server' | 'client'): string {
+export function getUserConfigFile(projectDir: string, platform: 'server' | 'client' | 'edge'): string | undefined {
```

Comment:
does this mean users have to define a `sentry.edge.config.js`?

we have to update the docstring if this is the case.

Reply:
> does this mean users have to define a `sentry.edge.config.js`?

They don't *have* to in the sense that something will crash, but they should if they want to capture events. We're even logging a warning that they should add the file in case they're using edge stuff.

> we have to update the docstring if this is the case.

Thanks for pointing that out. I updated the docstring.